app/services/get_albums.js

```diff
@@ -0,0 +1,7 @@
+const axios = require('axios');
+
+const endpoint = 'https://jsonplaceholder.typicode.com/albums';
+
+exports.getAlbums = () => axios.get(endpoint);
```

Comment:
Add some catch here for map some API error, if you don't do that an ugly 5xx would be sent to client